edge-apps/clock/screenly.yml

```diff
@@ -0,0 +1,42 @@
+app_id: ''
+root_asset_id: ''
+user_version: 'v1.0.0'
+revision: 1
+description: 'Screenly Clock Edge App'
+icon: ''
+author: 'Screenly, Inc.'
+homepage_url: ''
+settings:
+  override_timezone:
```

Comment:
This can be removed in favor of just `timezone` (or alternatively, merge the two and call them `override_timezone`.